src/mpl/configure.ac

```diff
@@ -1011,6 +1011,23 @@ if test "$have_gpu" = "no" ; then
     fi
 fi
 AM_CONDITIONAL([MPL_HAVE_ZE],[test "X${pac_have_ze}" = "Xyes"])
+
+# Check HIP availability
+if test "$have_gpu" = "no" ; then
+        PAC_CHECK_HEADER_LIB_OPTIONAL([hip],[hip_runtime_api.h],[hiprt],[hipStreamSynchronize])
+        if test "$pac_have_hip" = "yes" ; then
+            # also require -lhip (in addition to -lhiprt)
+            PAC_CHECK_HEADER_LIB([hip/hip_runtime.h],[hip],[hipMemGetAddressRange], pac_have_hip=yes, pac_have_hip=no)
```

Comment:
Is this required? I did not see `libhip.so` or `libhiprt.so` in my HIP installation. All I got is `libamdhip64.so`.

Reply:
yeah, this was also changed.